react-dataflow-dynamic/README.md

```diff
@@ -0,0 +1,62 @@
+# Observable Example: React + Dataflow (Dynamic Import)
+
+See it live: https://observablehq.github.io/examples/react-dataflow-dynamic/
+
+This modifies the React code generated by the Embed tool to facilitate dataflow between the notebook and the React app. The `height` state of `<App>` (React) is passed down to `<BrushableScatterplot>` (Observable), where it is passed into the Runtime’s `module.redefine` method. And a `setSelection` function is passed into the scatterplot component, which it calls in a custom [Observer](https://github.com/observablehq/runtime#observers) when the `selection` variable is fulfilled.
+
+This version imports the Observable Runtime and the notebook dynamically, so that you’ll always see the latest published version of the notebook when you load the page, instead of having to deploy a new version of your application. This also sidesteps the issues some bundlers have with Observable file attachments. Contrast with [static import](https://github.com/observablehq/examples/tree/main/react-dataflow/).
+
+```js
+const {Runtime, Inspector} = await import("https://cdn.jsdelivr.net/npm/@observablehq/runtime@4/dist/runtime.js");
```

Comment:
I guess the user can choose to install the runtime or to use this... can we perhaps have a comment here that suggests what to change if they want to install the runtime?

Reply:
yeah true